cmd/containerd-shim/main_unix.go

```diff
@@ -46,7 +46,7 @@ func init() {
 	flag.BoolVar(&debugFlag, "debug", false, "enable debug output in logs")
 	flag.StringVar(&namespaceFlag, "namespace", "", "namespace that owns the shim")
 	flag.StringVar(&socketFlag, "socket", "", "abstract socket path to serve")
-	flag.StringVar(&addressFlag, "address", "", "grpc address back to main containerd")
+	flag.StringVar(&addressFlag, "address", "", "ttrpc address back to main containerd")
```

Comment:
this stays grpc because its the address of containerd

Reply:
Woops.